enterprise_subsidy/apps/subsidy/api.py

```diff
@@ -90,6 +90,7 @@ def can_redeem(subsidy, lms_user_id, content_key):
       )
     """
     existing_transaction = subsidy.get_committed_transaction_no_reversal(lms_user_id, content_key)
+    is_active = subsidy.is_active
```

Comment:
Since the is_active field is not part of the model I would have expected you to calculate the is_active flag here and not just assign it? I don't think this will work.

Reply:
Hamzah set me straight:
> Ahh the is_active field is computed against the expiration date. There was no need to recompute it again, just assign a value based on the subsidy. If it’s true or false return that value